src/openforms/js/components/admin/form_design/FormStepDefinition.js

```diff
@@ -373,6 +399,32 @@ ConfigurationErrors.propTypes = {
   errors: PropTypes.arrayOf(PropTypes.arrayOf(PropTypes.string)),
 };
 
+const getDuplicatedComponents = componentNameSpace => {
+  let keyList = [];
+  let duplicates = [];
+
+  componentNameSpace.forEach(component => {
+    if (!keyList.includes(component.key)) {
+      keyList.push(component.key);
+      return;
+    }
+
+    if (!duplicates.some(duplicate => duplicate.key === component.key)) duplicates.push(component);
+  });
+
+  return duplicates;
+};
+
+const getComponentLocations = (formSteps, component) => {
+  return formSteps
+    .filter(step => step.configuration.components)
+    .map(step => {
+      if (step.configuration.components.some(stepComponent => component.key == stepComponent.key))
+        return step;
+    })
+    .filter(location => location);
```

Comment:
```suggestion
    .filter(Boolean);
```
accomplishes the same and makes it look less like an incomplete lambda :)